src/components/SafeListSidebar/SafeList/index.tsx

```diff
@@ -70,22 +71,28 @@ export const SafeList = ({ onSafeClick }: Props): ReactElement => {
   const classes = useStyles()
   const networks = getNetworks()
   const currentSafeAddress = extractSafeAddress()
-  const loadedSafes = useSelector(sortedSafeListSelector).filter(isNotLoadedViaUrl)
+  const loadedSafes = useSelector(sortedSafeListSelector)
+    .filter(isNotLoadedViaUrl)
+    .map((loadedSafe) => ({ ...loadedSafe, address: checksumAddress(loadedSafe.address) }))
   const ownedSafes = useOwnerSafes()
   const localSafes = useLocalSafes()
 
   return (
     <StyledList>
       {networks.map(({ id, backgroundColor, textColor, label }) => {
         const isCurrentNetwork = id === getNetworkId()
-        const localSafesOnNetwork = localSafes[id]
         const ownedSafesOnNetwork = ownedSafes[id] || []
-        const shouldExpandOwnedSafes =
-          (isCurrentNetwork &&
-            ownedSafesOnNetwork.some(
-              (address) => address === currentSafeAddress && !isSafeAdded(loadedSafes, address),
-            )) ||
-          (!localSafesOnNetwork.length && ownedSafesOnNetwork.length <= MAX_EXPANDED_SAFES)
+
+        const localSafesOnNetwork = localSafes[id].map((localSafe) => ({
+          ...localSafe,
+          address: checksumAddress(localSafe.address),
+        }))
```

Comment:
Should we also filter them by `isNotLoadedViaUrl`?

Reply:
The flag is not present in the `localSafe` object.

Reply:
The Safe is only saved locally after adding the Safe either way.

Reply:
Why did they appear for Martin then?

Reply:
Apparently older Safe releases automatically added Safes. I will double-check all of this, however.